src/overlays/actors/ovl_Obj_Hariko/z_obj_hariko.h

```diff
@@ -10,7 +10,10 @@ typedef void (*ObjHarikoActionFunc)(struct ObjHariko*, GlobalContext*);
 typedef struct ObjHariko {
     /* 0x0000 */ Actor actor;
     /* 0x0144 */ ObjHarikoActionFunc actionFunc;
-    /* 0x0148 */ char unk_148[0x10];
+    /* 0x0148 */ f32 bobbleStep;
+    /* 0x014C */ Vec3s headRotation;
+    /* 0x0152 */ s16 headOffset;
+    /* 0x0154 */ s16 unk154;
```

Comment:
```suggestion
    /* 0x0154 */ s16 unk_154; // Set but not used
```
Usually we do this in this situation.